emperor/support_files/js/emperor.js

```diff
@@ -183,6 +183,23 @@ function toggleScaleCoordinates(element) {
       operation(g_plotTaxa[index].scale.x, g_fractionExplained[g_viewingAxes[0]]),
       operation(g_plotTaxa[index].scale.y, g_fractionExplained[g_viewingAxes[0]]),
       operation(g_plotTaxa[index].scale.z, g_fractionExplained[g_viewingAxes[0]]));
+
+    //scale the dimensions of the taxa vectors
+    g_elementsGroup.remove(g_plotTaxaArrows[index])
+    g_mainScene.remove(g_plotTaxaArrows[index]);
+
+    var whiteColor = new THREE.Color();
+    whiteColor.setHex("0xFFFFFF");
+    var taxaVector = makeLine([0, 0, 0,],
+			      [g_plotTaxa[index].position.x,
+			       g_plotTaxa[index].position.y,
+			       g_plotTaxa[index].position.z],
+			      whiteColor, 2);
+
```

Comment:
Hmmm, I just pulled down your code into my machine and it shows up exactly the same way as it's showing on GitHub:

![screen shot 2015-03-31 at 2 45 39 pm](https://cloud.githubusercontent.com/assets/375307/6930041/a81bbc08-d7b4-11e4-979f-318ce4fd4090.png)